openshift/postgresql.dc.json

```diff
@@ -303,6 +336,24 @@
                       }
                     }
                   },
+                  {
```

Comment:
You don't need to inject the secrets as environment variable to the database container

Reply:
not currently, however i was hoping that it would be a bash script on the postgres container that would create the nris user from the secret, if that is not the long term solution then i suppose this could be removed.

Reply:
Hmm 🤔 Okay yeah that makes sense. Lets ship that and we can then find out if we use it later or not 👍